src/NuGet/Microsoft.CodeAnalysis.CSharp.CodeStyle.nuspec

```diff
@@ -36,5 +36,11 @@
     <!-- Scripts -->
     <file src="..\..\src\Setup\PowerShell\install.ps1" target="tools\" />
     <file src="..\..\src\Setup\PowerShell\uninstall.ps1" target="tools\" />
+
+    <!-- Satellite assemblies -->
+    <file src="Dlls\CSharpCodeStyle\**\Microsoft.CodeAnalysis.CSharp.CodeStyle.resources.dll" target="analyzers\dotnet\cs" />
```

Comment:
Nit: is it better to keep these by the lines higher up so it's easier to see that each regular one has it's resources along with?